### ISSUE TITLE
Quick question..

### ISSUE THREAD

**Issue by Holicsz** [Kiloutre/TekkenMovesetExtractor#73]
Can the extractor work with cracked/pirated version of Tekken 7?

**Comment by paidmm** [comment]
Yes but it probably only works with 1.0.32.34 Version. And don't forget to run the moveset extractor with Administrator
